lumodon_questions.md

```diff
@@ -0,0 +1,83 @@
+# Lumodon's Questions:
+========== Chapter 2: Firstclass ==============
+[Chapter 2: Firstclass](https://drboolean.gitbooks.io/mostly-adequate-guide/content/ch2.html#why-favor-first-class)
+
+```
+// go back to every httpGet call in the application and explicitly pass err
+// along.
+httpGet('/post/2', function(json, err) {
+  return renderPost(json, err);
+});
+```
+
+becomes 
+
+```
+// renderPost is called from within httpGet with however many arguments it wants
+httpGet('/post/2', renderPost);
+```
+### My Question: 
+- but comment doesn't make sense. Where is renderPost declared? What about when you're doing something different each time you render post? i.e. you're using anonomyous functions, so you must be doing something different with each httpGet call.
```

Comment:
A valid question, there's a lot left unsaid and assumed in this section. 

renderPost is not declared in this example; it's just an assumed function declared offscreen, a function to render a post. 

You're right that each httpGet cal does something different, in this case, httpGet takes in a URI, as well as a function that can do something about it.